test/shared/LiquidityOperations.sol

```diff
@@ -142,9 +129,13 @@ abstract contract LiquidityOperations is CommonBase {
         pure
         returns (bytes memory)
     {
-        Planner.Plan memory planner = Planner.init();
-        planner = planner.add(Actions.BURN, abi.encode(tokenId, config, hookData));
+        Plan memory planner = Planner.init();
+        planner.add(Actions.BURN_POSITION, abi.encode(tokenId, config, hookData));
         // Close needed on burn in case there is liquidity left in the position.
         return planner.finalize(config.poolKey);
     }
+
+    function snapLastDelta() internal view returns (BalanceDelta delta) {
+        delta = hook.deltas(hook.getDeltasLength() - 1); // just want the most recetly written to delta
+    }
```

Comment:
nit: imo a better name is `getLastDelta`

Reply:
also I think this (and `HookSavesDelta hook;`) belongs in `PosmTestSetup.sol`

Reply:
yeah agreed `snap` we use for gas snapshots - definitely prefer `getLastDelta`

Reply:
ah yes good catch, originally had it in LiquidityOperations cause I was referencing it but will change